src/pages/bill/[id].tsx

```diff
@@ -28,4 +28,4 @@ const PostViewPage: NextPageWithLayout = () => {
   );
 };
 
-export default PostViewPage;
+export default BillViewPage;
```

Comment:
Also in src/server/routers/bill.ts
`export const postRouter = createRouter()` should be `billRouter` 🙏